sktree/tree/unsupervised/_unsup_criterion.pyx

```diff
@@ -398,3 +396,159 @@ cdef class TwoMeans(UnsupervisedCriterion):
         self.n_node_samples = end - start
         self.start = start
         self.end = end
+
+cdef class FastBIC(UnsupervisedCriterion):
+    """Placeholder for FastBIC docs
+    """
+
+    cdef double sum_of_squares(
```

Comment:
I see! I thought the same as I was adapting `sum_of_squares` from TwoMeans. I created an issue (#17) to keep track, so that this gets done in an organized manner